tests/wicked/startandstop/sut/t12_vlan_xml_config.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: VLAN - Create a VLAN from wicked XML files
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+
+use base 'wickedbase';
+use strict;
+use testapi;
+use network_utils qw(iface ifc_exists);
+
+sub run {
+    my ($self) = @_;
+    my $config = '/etc/wicked/ifconfig/vlan.xml';
+    $self->get_from_data('wicked/xml/vlan.xml', $config);
+    my $local_ip = $self->get_ip(type => 'vlan', netmask => 1);
+    $local_ip =~ s'/'\\/';
+    my $iface = iface();
+    assert_script_run("sed 's/ip_address/$local_ip/' -i $config");
+    assert_script_run("sed 's/iface/$iface/' -i $config");
+    script_run("cat $config");
```

Comment:
I would do assert, just in case $config does not exist if the previous command failed.

Reply:
sed should throw error one line above in case, right ? Also I would like to leave this to separate PR . we have this pattern in several places ( sed'ing file and then cat the result ) I am about to extract to separate generic function. We can discuss this there

Reply:
ok, fine with me :)